<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="9673" started-at="2020-12-27T17:08:39Z" finished-at="2020-12-27T17:08:48Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="9673" started-at="2020-12-27T17:08:39Z" finished-at="2020-12-27T17:08:48Z">
      <class name="java_API.topic_01_checkenv">
        <test-method status="PASS" signature="beforeClass()[pri:0, instance:java_API.topic_01_checkenv@4461c7e3]" name="beforeClass" is-config="true" duration-ms="8663" started-at="2020-12-28T00:08:39Z" finished-at="2020-12-28T00:08:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method status="FAIL" signature="checkURL_and_title()[pri:0, instance:java_API.topic_01_checkenv@4461c7e3]" name="checkURL_and_title" duration-ms="43" started-at="2020-12-28T00:08:47Z" finished-at="2020-12-28T00:08:47Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected:<Guru99 Bank Home Page> but was:<Guru99 Bank Homennn Page>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected:<Guru99 Bank Home Page> but was:<Guru99 Bank Homennn Page>
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:101)
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:108)
	at java_API.topic_01_checkenv.checkURL_and_title(topic_01_checkenv.java:22)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkURL_and_title -->
        <test-method status="PASS" signature="afterClass()[pri:0, instance:java_API.topic_01_checkenv@4461c7e3]" name="afterClass" is-config="true" duration-ms="943" started-at="2020-12-28T00:08:47Z" finished-at="2020-12-28T00:08:48Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
      </class> <!-- java_API.topic_01_checkenv -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
